charts/argo-workflows/Chart.yaml

```diff
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Initialize Changelog"
+    - "[Fixed]: Controller ClusterRole to work with InstanceID"
```

Comment:
As I understand from https://artifacthub.io/docs/topics/annotations/helm/, you should replace the old value of the annotation, not append to it.

Reply:
Previously I've misunderstood how it works, fixed.